projects/plugins/jetpack/_inc/client/my-plan/my-plan-header/index.js

```diff
@@ -72,7 +72,12 @@ class MyPlanHeader extends React.Component {
 				if ( ! isInTheFuture( purchase.expiry_date ) && purchase.expiry_date !== null ) {
 					activation = <ProductActivated key="product-expired" type="product-expired" />;
 				} else {
-					activation = <ProductActivated key="product-activated" />;
+					activation = (
+						<ProductActivated
+							key="product-activated"
+							type={ purchase.expiry_date === null ? 'never-expires' : '' }
```

Comment:
Looks like there is a similar display issue reported here: 8267-gh-Automattic/jpop-issues

I _think_ this PR should handle that null expiry_date as well, but haven't tested as the PR build failed. Feel free to tag me for a review once this one is ready 👍

Reply:
Thanks for pointing out. I wasn't able to reproduce it, but will keep digging.

Reply:
I was able to see the issue with one of my old test sites, and looks like this PR has helped.

Before:

![Screenshot 2023-05-08 at 3 18 18 PM](https://user-images.githubusercontent.com/15803018/236937490-90b71401-000d-4467-8e06-434ce033894f.png)

After:

![Screenshot 2023-05-08 at 3 18 58 PM](https://user-images.githubusercontent.com/15803018/236937621-4e5286e7-4cb3-4252-b7a8-e613c9854b6b.png)